### PR TITLE
Add full danish translation

### PR DESCRIPTION
**Pull request by COHL7500**
# Describe your PR

Added full danish translation.

# Testing

No testing has been performed, as I was asked to just do a PR. I scrutinized the translation and it appears to be free of spelling erorrs/typos. If any issues will arise, I will of course perform proper testing and fix it :)

### CODE REVIEW

**Comment by COHL7500** [comment]
> archinstall/locales/base.pot

What is the issue? Could you elaborate if I am missing something? 

Thanks in advance and have a good day

**Comment by COHL7500** [comment]
@Torxed 

Is something remiss? Anything wrong with the PR since there's no approval?

**Comment by dylanmtaylor** [comment]
> > archinstall/locales/base.pot
> 
> What is the issue? Could you elaborate if I am missing something?
> 
> Thanks in advance and have a good day

This account is very likely a spam not. It's evident if you look at their other 'contributions'

**Comment by COHL7500** [comment]
> > > archinstall/locales/base.pot
> > 
> > 
> > What is the issue? Could you elaborate if I am missing something?
> > Thanks in advance and have a good day
> 
> This account is very likely a spam not. It's evident if you look at their other 'contributions'

@dylanmtaylor Oh, I see. Thanks for the clarification :)

File: archinstall/locales/base.pot [hunk 1]
```diff
@@ -784,3 +784,6 @@ msgstr ""
 
 msgid "Delete subvolume"
 msgstr ""
+
+msgid "Configured {} interfaces"
+msgstr ""
```
Comment (JoelLarson) [inline archinstall/locales/base.pot:789]: Why does this PR require a new `msgid` in all the languages? Was this something that was forgotten with a previous commit?

**Comment by JoelLarson** [comment]
There are a lot of changes in the `ru` translations. Was this done automatically by a linting tool?

File: archinstall/locales/base.pot (hunk 1 above)
Comment (COHL7500) [reply archinstall/locales/base.pot:789]: I'm not sure why. I used Poedit to translate the text and followed the steps provided in the README file within locales. 

I did notice this `msgid` suddenly became a requirement, though I presumed it was the work of the `./locales_generator.sh`.

**Comment by COHL7500** [comment]
@JoelLarson 
> There are a lot of changes in the `ru` translations. Was this done automatically by a linting tool?

I didn't touch the russian translation. As aforementioned, I just followed the exact steps mentioned in the README file in locales.

**Comment by COHL7500** [comment]
Perhaps I accidentally uncommented this `msgid`. for `Configured {} interfaces`. Should I comment it out?

**Comment by JoelLarson** [comment]
It looks like you have an added msgid/msgstr in each of the language files, then a bunch of string concatenations removed in the `ru` file. If you didn't add these (read: automated tooling did), then I can take a look and see what is causing the PR to look like this.

**Comment by COHL7500** [comment]
> It looks like you have an added msgid/msgstr in each of the language files, then a bunch of string concatenations removed in the `ru` file. If you didn't add these (read: automated tooling did), then I can take a look and see what is causing the PR to look like this.

@JoelLarson I only consciously made changes in the base.po of the danish translation, as aforementioned. I would very much be grateful if you could look into it or just tell me whatever I should fix. I will fix it ASAP and also attempt to test the translation if needed.

Thanks in advance and have a nice day :)

**Comment by COHL7500** [comment]
> @COHL7500 I've created a PR to update the locale files. Once that is merged, you can rebase on origin/master and your PR will be simplified to just the danish folder.

@JoelLarson Much appreciated. I will keep myself updated on your PR and follow your instructions.

**Comment by svartkanin** [comment]
Would you mind rebasing this PR :)

**Comment by svartkanin** [comment]
@COHL7500 apologies for the long delay, PRs are getting merged now but due to the large volume there's some conflicts happening. I'd like to merge this translation in, would you mind resolving the conflicts

**Comment by Torxed** [comment]
I would re-submit this PR with *only* the danish language files added via `git add archinstlal/locales/da/LC_MESSAGES/base.po` instead of adding all files affected by Poedit.

**Pull request closed without merging**